buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -1,5 +1,28 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run Begin.sh",
+      "timeoutInMinutes": 0,
+      "condition": "succeeded()",
+      "refName": "TaskRunBegin",
+      "task": {
+        "id": "10f1f9a1-74b0-47ab-87bf-e3c9c68e8b0d",
+        "versionSpec": "0.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "type": "InlineScript",
+        "scriptPath": "",
+        "args": "",
+        "cwd": "",
+        "failOnStandardError": "false",
+        "script": "if [ -f \"Begin.sh\" ]; then ./Begin.sh ; else echo \"Begin.sh script does not exist. Moving on.\" ; fi"
```

Comment:
What about AgentToolsPath?

Reply:
I still think you need to take AgentToolsPath into account like with the ps1 otherwise we will not be able to control where these scripts are on the agent machines.